### PR TITLE
git-authors: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #5137

### CODE REVIEW

File: pages/common/git-authors.md
Comment (bl-ue) [inline pages/common/git-authors.md:4]:
```suggestion
> Part of `git-extras`.
```

File: pages/common/git-authors.md
Comment (bl-ue) [inline pages/common/git-authors.md:7]: I tried it; this command does a bit more:
```suggestion
- Create a new (or update the existing) `AUTHORS` file in the current directory, populate it from the current Git repository, and open it in the default editor:
```

File: pages/common/git-authors.md
Comment (bl-ue) [inline pages/common/git-authors.md:11]:
```suggestion
- Display the full authors list for the current Git repository:
```

File: pages/common/git-authors.md
Comment (bl-ue) [inline pages/common/git-authors.md:15]:
```suggestion
- Display the full authors list for the current Git repository, excluding emails:
```

File: pages/common/git-authors.md
Comment (bl-ue) [inline pages/common/git-authors.md:3]:
```suggestion
> Display list of commiters of a Git repository, sorted by author's commit count.
```

File: pages/common/git-authors.md
Comment (CleanMachine1) [reply pages/common/git-authors.md:7]: I really did wonder about that file, I thought it was the command editing the AUTHORS file, turns out it didn't exist!

Thanks for that

**Comment by CleanMachine1** [comment]
Take a look now and give another review, I feel this is how it should be. 

Also the the usage of display and print is used since the display to me indicates that it isn't a firm output, rather something that can be scrolled through, while the print is used to indicate that it just dumps the entire lot, as if it was concatenated using cat

File: pages/common/git-authors.md
Comment (bl-ue) [inline pages/common/git-authors.md:5]: Unfortunately, we can't have more thant 3 total description lines (including the link) :/
```suggestion
> Display log or list of committers of a Git repository. Sorted by amount of commits.
> Part of `git-extras`.
```
The `AUTHORS` line isn't even true for all possible invocations of the command (running it with `--list` doesn't operate on `AUTHORS`, I don't think)

See https://github.com/tldr-pages/tldr/pull/5807#discussion_r617511072 for details.

Comment (CleanMachine1) [reply pages/common/git-authors.md:5]: You're right, the list doesn't use a file

Comment (CleanMachine1) [reply pages/common/git-authors.md]: take a look at the commit after this

**Comment by CleanMachine1** [comment]
@navarroaxel @marchersimon @bl-ue Please take a look at this PR and give your thoughts so we can work on getting it merged

File: pages/common/git-authors.md
Comment (marchersimon) [inline pages/common/git-authors.md:3]:
```suggestion
> Display log or list of committers of a Git repository.
```
This doesn't really seem important.

File: pages/common/git-authors.md
Comment (marchersimon) [inline pages/common/git-authors.md:7]:
```suggestion
- Display a log of committers in the default editor, reading from the `AUTHORS` file:
```

File: pages/common/git-authors.md
Comment (marchersimon) [inline pages/common/git-authors.md:11]:
```suggestion
- Print a full list of committers:
```
I don't think that's neccecary since it's the only way it makes sense.

File: pages/common/git-authors.md
Comment (marchersimon) [inline pages/common/git-authors.md:15]:
```suggestion
- Display a log of committers, exluding emails, in the default editor, reading from the `AUTHORS` file:
```

**Review by marchersimon — CHANGES_REQUESTED** [review]

File: pages/common/git-authors.md
Comment (CleanMachine1) [reply pages/common/git-authors.md:3]: It does to me, since otherwise someone will be confused when a new random file appears in their directory

File: pages/common/git-authors.md
Comment (CleanMachine1) [reply pages/common/git-authors.md:7]: The editor is required since the list version doesn't

File: pages/common/git-authors.md
Comment (CleanMachine1) [reply pages/common/git-authors.md:11]: Ok yes

File: pages/common/git-authors.md
Comment (CleanMachine1) [reply pages/common/git-authors.md:15]: why does this have the default editor bit but the other suggestion doesn't?
Consistency isn't present here

File: pages/common/git-authors.md
Comment (marchersimon) [reply pages/common/git-authors.md:7]: I haven't removed it 🤔

Comment (CleanMachine1) [reply pages/common/git-authors.md:7]: so you haven't

Comment (CleanMachine1) [reply pages/common/git-authors.md:7]: ok then commited

File: pages/common/git-authors.md
Comment (marchersimon) [reply pages/common/git-authors.md:3]: How about that? (We should avoid `you`, `yours`, etc.)
```suggestion
> Display log or list of committers of a Git repository. Everything except `--list` will create a new file called `AUTHORS`.
```

**Review by marchersimon — APPROVED** [review]

File: pages/common/git-authors.md
Comment (navarroaxel) [inline pages/common/git-authors.md:11]:
```suggestion
- Display a full list of committers:
```

File: pages/common/git-authors.md
Comment (navarroaxel) [inline pages/common/git-authors.md:15]: Why log?
```suggestion
- Display a full list of committers, excluding emails, in the default editor, reading from the `AUTHORS` file:
```
(a typo here)

File: pages/common/git-authors.md
Comment (marchersimon) [reply pages/common/git-authors.md:11]: > Also the the usage of display and print is used since the display to me indicates that it isn't a firm output, rather something that can be scrolled through, while the print is used to indicate that it just dumps the entire lot, as if it was concatenated using cat

I kind of have to agree with @CleanMachine1 here

File: pages/common/git-authors.md
Comment (marchersimon) [reply pages/common/git-authors.md:15]: Good question. It is indeed only a plain list.

Comment (CleanMachine1) [reply pages/common/git-authors.md:15]: < Why log? 

because the list is used in the list variant of the command.

Comment (navarroaxel) [reply pages/common/git-authors.md]: > because the list is used in the list variant of the command.

I didn't get this.

File: pages/common/git-authors.md
Comment (CleanMachine1) [reply pages/common/git-authors.md:11]: It is print since it dumps it, display to me indicates more of a elegant way of displaying.
the display indicates that it is opened in a text editor

File: pages/common/git-authors.md
Comment (marchersimon) [reply pages/common/git-authors.md]: But it's also a plain list when you run `git authors`.

Comment (CleanMachine1) [reply pages/common/git-authors.md]: For me it opens in VIM, since it opens in the text editor

Comment (CleanMachine1) [reply pages/common/git-authors.md]: --list makes a dumped list

Comment (navarroaxel) [reply pages/common/git-authors.md]: I think a `log` is a record of events, in this case were are not talking about events of any kind.

File: pages/common/git-authors.md
Comment (CleanMachine1) [reply pages/common/git-authors.md:11]: > > Also the the usage of display and print is used since the display to me indicates that it isn't a firm output, rather something that can be scrolled through, while the print is used to indicate that it just dumps the entire lot, as if it was concatenated using cat
> 
> I kind of have to agree with @CleanMachine1 here

couldn't find my reply! Was looking for it

File: pages/common/git-authors.md
Comment (marchersimon) [reply pages/common/git-authors.md]: But what does that have to do with a log? To me a log is something, where changes or events are saved with a timestamp, like `git log {{file}}`.

Comment (CleanMachine1) [reply pages/common/git-authors.md]: To me a log, is a list of events as well, however it isn't required to be timestamped

Comment (marchersimon) [reply pages/common/git-authors.md]: `git authors` only outputs a list of contributors, which I wouldn't call a log, but just a list.

Comment (CleanMachine1) [reply pages/common/git-authors.md]: What happens exactly, the same as --list with it?

**Comment by CleanMachine1** [comment]
@bl-ue @navarroaxel This is ready for review

File: pages/common/git-authors.md
Comment (navarroaxel) [reply pages/common/git-authors.md]: `--list` don't create the `AUTHORS` file.

File: pages/common/git-authors.md
Comment (navarroaxel) [inline pages/common/git-authors.md:15]: This is not a log of events, is just a list.
```suggestion
- List committers, excluding emails, in the default editor, reading from the `AUTHORS` file:
```

File: pages/common/git-authors.md
Comment (navarroaxel) [inline pages/common/git-authors.md:7]: This command create or update the `AUTHORS` file:
```suggestion
- Create or replace the `AUTHORS` file with the list of committers and open it in the default editor:
```

**Review by bl-ue — APPROVED** [review]

File: pages/common/git-authors.md
Comment (sbrl) [inline pages/common/git-authors.md:3]: Perhaps this will read better?
```suggestion
> Display a log or list of committers of a Git repository. Everything except `--list` will create a new file called `AUTHORS`.
```

File: pages/common/git-authors.md
Comment (sbrl) [inline pages/common/git-authors.md:15]: Perhaps this will read better?
```suggestion
- Launch the default editor to display a list of committers (excluding emails) from the `AUTHORS` file:
```

**Review by sbrl — COMMENTED** [review]
Thanks for the new page! I've left some comments below for you to review.

File: pages/common/git-authors.md
Comment (CleanMachine1) [reply pages/common/git-authors.md:15]: I disagree with this one, it sounds wrong

Comment (navarroaxel) [reply pages/common/git-authors.md:15]: Maybe sounds wrongs, but @sbrl is right, is not a list in terminal, Launch an app with something. :thinking: @bl-ue ?

Comment (bl-ue) [reply pages/common/git-authors.md:15]: To me they both imply basically the same meaning.

Comment (marchersimon) [reply pages/common/git-authors.md:15]: I think too. When you list something in the default editor, I think that implies that it launches it first. I prefer the first one.

**Review by navarroaxel — APPROVED** [review]

**Comment by sbrl** [comment]
Erm why did you merge this @CleanMachine1? Just because you disagree with one of my comments doesn't mean that you can merge it without letting me reply.

**Comment by CleanMachine1** [comment]
Sorry about that, however it seemed that simon agreed with my original wording and bl-ue didn't mind.
Sorry, didn't mean for any harm or wrong doing

File: pages/common/git-authors.md
Comment (sbrl) [reply pages/common/git-authors.md:15]: Hrm, I see the concerns here. However this PR has been prematurely merged, which limits the usefulness of me suggesting anything else here. Regardless, what about this?
```suggestion
- Display a list of committers (excluding emails) from the `AUTHORS` file in the default text editor:
```

**Comment by sbrl** [comment]
Next time, if someone disagrees with something wait until they reply and *everyone* is in agreement before merging :-)

**Pull request merged**
